android/src/main/java/com/saranshmalik/rnzendeskchat/RNZendeskChat.java

```diff
@@ -206,14 +212,30 @@ public void openTicket(){
           .show(activity);
   }
 
+  @ReactMethod
+  public void hasOpenedTickets(final Promise promise){
+    requestProvider = Support.INSTANCE.provider().requestProvider();
+    requestProvider.getAllRequests(new ZendeskCallback<List<Request>>() {
+      @Override
+      public void onSuccess(List<Request> requests) {
+        // Handle success
+        promise.resolve(requests.size());
+      }
+      @Override
+      public void onError(ErrorResponse errorResponse) {
+        // Handle error
+        promise.resolve(-1);
```

Comment:
Perhaps we can get a more precise error message from `errorResponse`?

Reply:
Refactored in [8827d2d](https://github.com/pagopa/io-react-native-zendesk/pull/12/commits/8827d2d126a8e91bf43578065ce04e3a90b1f76b)